src/main/java/com/visenze/datatracking/Constants.java

```diff
@@ -0,0 +1,17 @@
+package com.visenze.datatracking;
+
+public class Constants {
+    public static final String BASE_URL = "https://staging-analytics.data.visenze.com/v3/";
+    public static final String PLATFORM = "Mobile";
+    public static final String OS = "Android";
+    public static final String SDK_NAME = "Android SDK";
+
+
+    public static class Action {
```

Comment:
for action these are the ones we will support at the beginning:
- product_click
- search 
- product_view (for impression)
- app_install (app)
- app_uninstall (app)
- transaction
- add_to_cart

product_click would require validation of product ID and so on as described in business PRD.

Reply:
pls add all of them

Reply:
ok, but how do we monitor app_uninstall action ?  app_install action. do you want app_install event to be send out when sdk first initialized ?

Reply:
No need to monitor app_install/app_uninstall actions. In the interface, customer needs to create Event which includes all required information). Probably better to convert this to an enum (see my code at com.visenze.analytics.v3.model.EventAction).

You just need to provide a few different methods e.g.

sendProductClick(String queryId, String pid, ..). see QP's documents for the main 5 events we need to support (product_click, product_view, search, add_to_cart, transaction)

Reply:
```
            //send multiple events
            EventData e1 = DataTracking.getInstance().createProductClickEvent("1234", "343", "url", 1);
            EventData e2 = DataTracking.getInstance().createViewEvent();
            EventData e3 = DataTracking.getInstance().createProductImpressionEvent("1234", "343", "url", 2);
            EventData e4 = DataTracking.getInstance().createAddCartEvent("1234", "343", "url", 4);
            EventData e5 = DataTracking.getInstance().createTransactionEvent("1234", "343", 34.2);

            List<EventData> events = new ArrayList<EventData> ();
            events.add(e1);
            events.add(e2);
            events.add(e3);
            events.add(e4);
            events.add(e5);
            DataTracking.getInstance().sendEvents(events);
``` 
the interface is like this. is that ok ?